src/go/util/httppattern/uri_template.go

```diff
@@ -62,7 +62,7 @@ type variable struct {
 	HasDoubleWildCard bool
 }
 
-func (u *UriTemplate) String() string {
```

Comment:
Please check all the place using uriTemplate as Stringer interface otherwise the log may be messed up. Or implement this method with ExactMatchString(false) just in cash.

Reply:
Nowhere else is using it, it would be a compile time error since Stringer is now longer implemented.

I reimplemented stringer as you suggested.